server/forge/bitbucket/bitbucket.go

```diff
@@ -18,10 +18,10 @@ package bitbucket
 import (
 	"context"
 	"fmt"
+	"golang.org/x/oauth2"
```

Comment:
imports got unsorted ... I'll push a fix tomorow ... or you are fist wit the fix ;)